antora.yml

```diff
@@ -1,5 +1,5 @@
-name: suse-manager
-title: SUSE Manager 4.3 Documentation
+name: uyuni
+title: Uyuni
```

Comment:
Not sure if this needs to be `Uyuni` or `Uyuni Documentation`.

@jcayouette?

Reply:
Leave as is, we have scripts that replace this content.